components/NfcShare.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div v-if="supportsNfc">
+    <button
+      v-if="!isSharing"
+      @click="writeNfc">
+      Share via NFC
+    </button>
+    <div v-if="isSharing">
+      Hold your device near a reader to share your credential.
+      <button @click="cancelWrite">
+        Cancel
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+import {defineComponent, ref} from 'vue';
+import {Notify} from 'quasar';
+
+export default defineComponent({
```

Comment:
This could be simplified to `export default {` which would then allow for defineComponent to not be imported.

See AddUserModal.vue component for reference